KubeArmor/feeder/feeder.go

```diff
@@ -562,7 +562,19 @@ func (fd *Feeder) PushMessage(level, message string) {
 
 // PushLog Function
 func (fd *Feeder) PushLog(log tp.Log) {
-	log = fd.UpdateMatchedPolicy(log)
+
+	if cfg.GlobalCfg.Enforcer == "bpflsm" && fd.Enforcer == "BPFLSM" && log.Enforcer != "BPFLSM" {
```

Comment:
```suggestion
	if cfg.GlobalCfg.EnforcerAlerts && fd.Enforcer == "BPFLSM" && log.Enforcer != "BPFLSM" {
```
It is possible that the user needs a BPFLSM enforcer but alerts from ebpf, so this configuration is for that.